_data/retail.yml

```
@@ -116,11 +137,15 @@ websites:
     tfa:
       - totp
     doc: https://www.digitec.ch/en/page/new-two-factor-authentication-for-your-user-account-6589
+    regions:
+      - ch
```

```suggestion
      - ch
      - li
```
Digitec also serves [Liechtenstein](https://www.digitec.ch/en/wiki/556).